File: PROGRAMMING/LIT-Python/strong number.py
no=int(input("enter a number: "))
record=no
sum=0
while no>0:
    last=no%10
    fact=1
    while last>0:
        fact=fact*last
        last-=1
    sum=sum+fact
    no=no//10

print(sum)
if sum==record:
    print("strong number")
else:
    print("not a strong number")

File: PROGRAMMING/LIT-C/strong number.c
#include"stdio.h"
main()
{
    int n,s=0,r,m;
    printf("enter a  number");
    scanf("%d", &n);
    m=n;
    while(n>0)
    {
        r=n%10;
        s=s+fact(r);
        n=n/10;
    }
    if(s==m)
        printf("strong number");
    else
        printf("not a stong number");
}
int fact(int n)
{
    int f=1;
    while(n>0)
    {
        f=f*n;
        n--;
    }
    return f;
}
